### DISCUSSION TITLE
`normalizeProps()` turns `style` attribute value to a string, but `mergeProps()` turns it back to an object (when using Svelte)

### DISCUSSION THREAD

**Discussion by babakfp** [chakra-ui/zag#2045] [General]
Hi

This is my code:

```ts
const mergedAttributes = $derived(
  normalizeProps.input(
    mergeProps(
      radioGroup.getItemHiddenInputProps(itemProps),
      htmlInputAttributes,
    ),
  ),
)
```

`radioGroup.getItemHiddenInputProps(itemProps)` returns the style attribute value as a string. When wrapping them with `mergeProps`, they turn into an object.

![image](https://github.com/user-attachments/assets/bd6b2262-1f78-40f3-ae1a-1a682cc65c5e)

NOTE: ignore the word `init`. It has something to do with Svelte reactivity.

Is this a bug? Because Svelte doesn't expect an object inside the `style` attribute and [this code](https://github.com/chakra-ui/zag/blob/1f2d9093f62570ae0673d7c870ce8bab76c4986a/packages/frameworks/svelte/src/normalize-props.ts#L55C14-L55C28) converts a style object into a string as expected, to make it work with Svelte.

Maybe the Svelte version of `mergeProps` needs to turn the value of `style` attribute back to a string at the end?

**Comment by anubra266** [comment]
@babakfp I'm unable to reproduce this. Could you attach a reproduction

**Reply by babakfp** [reply]
https://github.com/babakfp/zag-discussions-2045

Only related content is located at `src\routes\+page.svelte`

**Reply by anubra266** [reply]
Ok @babakfp I'm able to reproduce now. Could you open an issue in this regards. THanks

**Reply by babakfp** [reply]
Awesome! Thanks.

Link to issue: https://github.com/chakra-ui/zag/issues/2046
